Author: Joel Roth <joelz@pobox.com>
Description: Fix path to ecasoundc.h file to suit libecasoundc2.2-dev version 2.7.0-1.1

--- a/Ecasound.xs
+++ b/Ecasound.xs
@@ -2,7 +2,7 @@
 #include "perl.h"
 #include "XSUB.h"
 
-#include <libecasoundc/ecasoundc.h>
+#include <ecasoundc.h>
 
 MODULE = Audio::Ecasound        PACKAGE = Audio::Ecasound
 
